.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+dist: trusty
+
+language: cpp
+
+compiler:  gcc
+
+before_install:
+- sudo apt-get update -qq
+- sudo apt-get install -qq libperl4-corelibs-perl libtool xorg-dev doxygen python-dev alien swig libgtest-dev libstdc++6 libxml2-dev gettext libxinerama-dev libxft-dev libxrandr-dev libxcursor-dev libgdbm-dev libc6 libc6-dev libmng-dev zlib1g-dev libcap-dev libpng12-0 libpng12-dev freeglut3-dev flex libx11-dev bison++ bisonc++ libjpeg-dev libjpeg8-dev python2.7 python2.7-dev python2.7-psycopg2 libogdi3.2-dev libgif-dev libxerces-c-dev libgeos-dev libgeos++-dev libfreetype6 libfreetype6-dev python-imaging libproj-dev python-setuptools libgif-dev libxerces-c-dev libcap-dev libpq-dev openssl libxml2-utils libxmu-dev
+
+
+before_script:
+  - cd earth_enterprise
+
+script:
+  - scons -j3 release=1 build > build.log
+  - cd src/NATIVE-REL-x86_64/bin/tests
+  - ./RunAllTests.pl
+
+cache:
+  directories:
+    - $HOME/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/third_party
```

Comment:
I think I see part of the reason why the cache isn't working, it's actually specifying the wrong path. This is the path it's caching:
/home/travis/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/third_party
But, this is the actual path to where the third_party directory is being built:
/home/travis/build/tst-eclamar/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/third_party/

Reply:
But, to be honest, even if it was caching the right directory, I don't think it would work. I tried an experiment and the build fails because of missing files. So, I think you will need to cache a lot more directories than just this one. It may be better to create a custom docker image

Reply:
I suggest removing the cache for now. I suspect that it may actually be slowing us down instead of speeding us up. At any rate, it certainly isn't speeding anything up at the moment :)

Reply:
Ok. Will do.